x-pack/plugins/license_management/public/sections/license_dashboard/request_trial_extension/request_trial_extension.js

```diff
@@ -14,13 +14,13 @@ export const RequestTrialExtension = ({ shouldShowRequestTrialExtension }) => {
   }
   const description = (
     <span>
-      If you’d like to continuing using Security, Machine Learning, and our
+      If you’d like to continuing using security, machine learning, and our
       other awesome{' '}
       <EuiLink
         href="https://www.elastic.co/subscriptions/xpack"
         target="_blank"
       >
-        platinum features
+        Platinum features
```

Comment:
Why is Platinum capitalized?

Reply:
@tylersmalley Platinum refers to the name of a license.